support-frontend/assets/components/checkoutNudge/checkoutNudge.tsx

```diff
@@ -45,52 +46,50 @@ const top = css`
 const topheading = css`
 	margin-top: ${space[3]}px;
 `;
-const heading = (backColor: string, marginBottom: number) => css`
-	margin-left: 10px;
-	color: ${backColor};
-	${headline.xxsmall({ fontWeight: 'bold', lineHeight: 'tight' })};
-	line-height: 108%;
+const heading = (
+	backColor: string,
+	marginBottom: number,
+	shrinkHeadline?: boolean,
+) => css`
+	margin-left: 9px;
+	 ${headline.xxsmall({ fontWeight: 'bold', lineHeight: 'regular' })};
+  color: ${backColor};
 
-	${from.mobileMedium} {
-		margin-left: 12px;
+  ${from.mobileMedium} {
 		margin-bottom: ${marginBottom}px;
-		${headline.xsmall({ fontWeight: 'bold', lineHeight: 'tight' })};
-		line-height: 108%;
-	}
+    ${
+			shrinkHeadline
+				? headline.xxsmall({ fontWeight: 'bold', lineHeight: 'tight' })
+				: headline.xsmall({ fontWeight: 'bold', lineHeight: 'tight' })
```

Comment:
You could change the font-size like this:
```suggestion
			font-size: ${shrinkHeadline ?  ...px : ...px}
```

Reply:
Ah yes, much neater, have replaced with suggestion